src/com/TTT/TurnUI.java

```diff
@@ -10,40 +10,74 @@
     private BoardPrinter printer;
     private Scanner scanner;
     private String inputPrompt = "Please choose a space: ";
+    private int currentPlayer;
+    private String player1Marker;
+    private String player2Marker;
 
     public TurnUI(BoardPrinter printer, PrintStream out, InputStream in) {
         this.printer = printer;
         this.out = out;
         scanner = new Scanner(in);
         scanner.useDelimiter("\n");
+        currentPlayer =1;
+        player1Marker = PlayerMarkers.X.symbol();
+        player2Marker = PlayerMarkers.O.symbol();
     }
 
-    public void takeTurn(Board board, Game game) {
+    public void takeTurn(Board board) {
         String selectedSpace = getPlayerInput(board, inputPrompt);
-        String winningMarker = game.getPlayerMarker(game.currentPlayer());
+        String winningMarker = getPlayerMarker(currentPlayer);
 
-        while(!validSelection(selectedSpace) || !game.selectionOnBoard(parseSelection(selectedSpace)) || board.isOccupied(parseSelection(selectedSpace))) {
-            selectedSpace = getPlayerInput(board,  invalidReasonText(board, game, selectedSpace) + inputPrompt);
+        while(!validSelection(selectedSpace) || !selectionOnBoard(parseSelection(selectedSpace), board) || board.isOccupied(parseSelection(selectedSpace))) {
```

Comment:
As mentioned before, why not move these three calls to one call so the while loop reads `while(invalid(input))`?